subsys/logging/log_core.c

```diff
@@ -1338,13 +1355,28 @@ static void log_process_thread_func(void *dummy1, void *dummy2, void *dummy3)
 {
 	__ASSERT_NO_MSG(log_backend_count_get() > 0);
 
+#ifdef CONFIG_LOG_BACKEND_EVENT_NOTIFICATION_SUPPORT
```

Comment:
all ifdefs in log_core.c could be removed.

Reply:
Removed `#ifdef`.